oak_functions/sdk/test_utils/src/lib.rs

```diff
@@ -180,6 +189,75 @@ where
     }
 }
 
+/// Builds the crate identified by the given name as a Linux binary, and returns the path of the
```

Comment:
I find it always a bit hard to find out which "name" we to use and if there are any assumptions on it. I believe e.g., it has to be reachable from "WORKSPACE_ROOT" (maybe it is also dependent on our workspace setup?). 

Anyway: this would be an example for the name, or am I wrong?

https://github.com/project-oak/oak/blob/4228c6b87ce377cf8c595535dd5932c822170441/oak_functions_freestanding/Cargo.toml#L2 

So maybe "identified by "name" from Cargo.toml" would help? (Also below for `build_rust_crate_linux`)

Reply:
Done, though I think it should be clear what a crate name is. I also clarified the workspace relationship now, thanks!